conans/util/files.py

```diff
@@ -103,7 +103,8 @@ def decode_text(text, encoding="auto"):
 
 
 def touch(fname, times=None):
-    os.utime(fname, times)
+    with open(fname, 'a'):
```

Comment:
small fix. os.utime requires the created file first.

Reply:
This might be wrong: https://stackoverflow.com/questions/42630281/utime-has-no-effect-in-windows

You need to close first the file handle, then utime it